ZWaveLib/CommandClasses/ClimateControlSchedule.cs

```diff
@@ -0,0 +1,109 @@
+﻿/*
+    This file is part of ZWaveLib Project source code.
+
+    ZWaveLib is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    ZWaveLib is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with ZWaveLib.  If not, see <http://www.gnu.org/licenses/>.  
+*/
+
+/*
+ *     Author: Generoso Martello <gene@homegenie.it>
+ *             Ben Voss
+ *     Project Homepage: https://github.com/genielabs/zwave-lib-dotnet
+ */
+
+using System.Collections.Generic;
+
+namespace ZWaveLib.CommandClasses
+{
+    public class ClimateControlSchedule : ICommandClass
+    {
+        public CommandClass GetClassId ()
+        {
+            return CommandClass.ClimateControlSchedule;
+        }
+
+        public NodeEvent GetEvent (ZWaveNode node, byte [] message)
+        {
+            if (message.Length == 0) {
+                return null;
+            }
+
+            byte cmdType = message [1];
+
+            if (cmdType == (byte)Command.ScheduleReport) {
+                var climateControlScheduleValue = ClimateControlScheduleValue.Parse (message);
+                return new NodeEvent (node, EventParameter.ClimateControlSchedule, climateControlScheduleValue, 0);
+            }
+
+            if (cmdType == (byte)Command.ScheduleChangedReport) {
+                return new NodeEvent (node, EventParameter.ClimateControlScheduleChanged, message[2], 0);
+            }
+
+            if (cmdType == (byte)Command.ScheduleChangedReport) {
```

Comment:
@ben-voss, I've just noted that you use `Command.ScheduleChangedReport` twice, looks like an error.